src/containers/app/index.js

```diff
@@ -14,23 +11,13 @@ const renderIfSignedIn = (element, props) => (props.isLoggedIn && element);
 const renderIfSignedOut = (element, props) => (!props.isLoggedIn && element);
 
 const App = (props) => (
-    <div>
-        <header>
-            <Link to="/">Home</Link>
-            { renderIfSignedIn(<Link to="/app">My Application</Link>, props) }
-            { renderIfSignedOut(<Link to={{ pathname: "/login" }}>Log in</Link>, props) }
-            { renderIfSignedOut(<Link to={{ pathname: "/register" }}>Register</Link>, props) }
-            { renderIfSignedIn(<Link to={{ pathname: "/logout" }}>Log out</Link>, props) }
-        </header>
-
-        <main>
-            <Route exact path="/" component={ Home } />
-            <Route exact path="/app" component={ Application } />
-            <Route exact path="/register" component={ Authentication } />
-            <Route exact path="/login" component={ Authentication } />
-            <Route exact path="/logout" component={ Logout }/>
-        </main>
-    </div>
```

Comment:
where are we planning to render the navigation bar on pages that are not the home page?

Reply:
do we need the navigation bar on other pages? the only place you would need to login button is on the home page, and the only time you would need to logout button is in the dashboard/application page, but tbh I'm not sure that people really need/would use the logout button

Reply:
there are going to be other links like `/dashboard` (which should show the status of your application), `/application`, and `/talks`. how would we navigate to those pages?